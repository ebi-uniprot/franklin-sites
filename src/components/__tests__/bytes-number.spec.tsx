import { render } from '@testing-library/react';

import { BytesNumber } from '../..';

describe('Bytes number component', () => {
  it('should render', () => {
    const { asFragment } = render(
      <BytesNumber decimals={1}>{2621388791}</BytesNumber>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
