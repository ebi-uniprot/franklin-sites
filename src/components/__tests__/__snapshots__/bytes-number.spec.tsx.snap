// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bytes number component should render 1`] = `
<DocumentFragment>
  2.4 GB
</DocumentFragment>
`;
